src/app/pages/HomePage/HomePage.tsx

```diff
@@ -1,87 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Header from '../../components/Header/Header';
 import GearCard from '../../components/GearCard/GearCard';
 import NavBar from '../../components/NavBar/NavBar';
+import type { Gear } from '../../../types';
 import style from './HomePage.module.css';
 
-type MockData = {
-  iconType: 'Synth' | 'Effect' | 'Drum';
-  name: string;
-  connections: { channel: string; connection: string }[];
-};
+export default function HomePage(): JSX.Element {
+  const [gear, setGear] = useState<Gear[]>([]);
 
-const GearCardMockData: MockData[] = [
-  {
-    iconType: 'Drum',
-    name: 'DrumBrute',
-    connections: [
-      { channel: 'Channel 3', connection: 'Output to' },
-      { channel: 'MacBook', connection: 'MIDI from' },
-      { channel: 'Port 13', connection: 'USB Port' },
-    ],
-  },
-  {
-    iconType: 'Effect',
-    name: 'Big Sky',
-    connections: [
-      { channel: 'Channel 12', connection: 'Output to' },
-      { channel: 'Port 13', connection: 'USB Port' },
-    ],
-  },
-  {
-    iconType: 'Synth',
-    name: 'Matriach',
-    connections: [
-      { channel: 'Channel 20', connection: 'Output to' },
-      { channel: 'MacBook', connection: 'MIDI from' },
-      { channel: 'DrumBrute', connection: 'MIDI to' },
-      { channel: 'Port 3', connection: 'USB Port' },
-      { channel: 'NDLR', connection: 'Input' },
-    ],
-  },
-  {
-    iconType: 'Drum',
-    name: '808',
-    connections: [
-      { channel: 'Channel 6', connection: 'Output to' },
-      { channel: 'MacBook', connection: 'MIDI from' },
-      { channel: 'Port 10', connection: 'USB Port' },
-      { channel: 'Sequencer', connection: 'Input from' },
-    ],
-  },
-  {
-    iconType: 'Drum',
-    name: 'DrumBrute',
-    connections: [
-      { channel: 'Channel 3', connection: 'Output to' },
-      { channel: 'MacBook', connection: 'MIDI from' },
-      { channel: 'Port 13', connection: 'USB Port' },
-    ],
-  },
-  {
-    iconType: 'Synth',
-    name: 'Matriach',
-    connections: [
-      { channel: 'Channel 20', connection: 'Output to' },
-      { channel: 'MacBook', connection: 'MIDI from' },
-      { channel: 'DrumBrute', connection: 'MIDI to' },
-      { channel: 'Port 3', connection: 'USB Port' },
-      { channel: 'NDLR', connection: 'Input' },
-    ],
-  },
-];
+  async function fetchGear() {
+    const response = await fetch('/api/gear', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    const gear = await response.json();
+    setGear(gear);
+  }
```

Comment:
Don't you have a `useFetch`?

Reply:
See [comment ](https://github.com/Seborider/plugs./pull/48#discussion_r715634354)below